src/client/src/models/Scene.js

```diff
@@ -108,22 +108,47 @@ const Scene = {
     Scene.setScenes(list)
   },
 
+
+
+
+
+
+
   // this function will save the current scene to the localStorage
   saveStart: function() {
     const list = Scene.getScenes()
 
     // this is a check to make sure the value entered into start is a number
     const start_check = Number(Scene.currentScene.start)
     if (isNaN(start_check)) {
```

Comment:
Once you've cleared the start of white space, it swill still be the empty string. So isNaN() will still return false because start_check == 0. So in this check, you need to then check if the cleansed start is the empty string.

Reply:
Done handled by `start_replace`.